mistletoe/html_renderer.py

```diff
@@ -73,24 +93,39 @@ def render_image(self, token: span_token.Image) -> str:
             title = ''
         return template.format(token.src, self.render_to_plain(token), title)
 
-    def render_link(self, token: span_token.Link) -> str:
-        template = '<a href="{target}"{title}>{inner}</a>'
+    def render_link(self, token: span_token.Link, attr: Attributes = None) -> str:
+        """
+        Renders the given `Link`.
+
+        :Parameters:
+            attr
+                The ``href`` attribute is set regardless of what attributes you pass.
```

Comment:
Single backticks in reST are used for interpreted text, documentation generators like [pydoctor](https://github.com/twisted/pydoctor) use it to link API members. Double backticks on the other hand are just inline code.

Note that your editor happens to be VS Code it just renders the docstrings as Markdown by default.

Reply:
OK, thanks a lot for your explanation. I find it generally quite difficult to find some quick all-in-one reference to this topic of docstrings / reST, possibly because it is far from being standardized...